users/users.go

```diff
@@ -8,8 +8,28 @@ import (
 
 	"github.com/abhishekhsheoran/library-management/models"
 	"github.com/abhishekhsheoran/library-management/utils"
+	"github.com/gorilla/mux"
+	"go.mongodb.org/mongo-driver/bson"
 )
 
+func verifyUserAttributes(user models.User) string {
+	var error string
+	if user.Name == "" {
+		error = "user name cannot be empty"
+		return error
```

Comment:
Make custom error here, the error returned is nil in this case. You can use fmt.Errorf() or errors.New() to create error with custom error message